methods/get.js

```diff
@@ -0,0 +1,12 @@
+const AWS = require('aws-sdk');
```

Comment:
Yeah, it returns the raw image data.  If it was uploaded with an s3 key name ending in .jpg/.png/.etc, your browser will interpret it as a valid image file and download it.